runtime/compiler/codegen/J9AheadOfTimeCompile.hpp

```diff
@@ -57,17 +57,26 @@ class OMR_EXTENSIBLE AheadOfTimeCompile : public OMR::AheadOfTimeCompileConnecto
    uint8_t* dumpRelocationHeaderData(uint8_t *cursor, bool isVerbose);
 
    /**
-    * @brief Initializes the common fields in the raw relocation record header. It then delegates
-    *        to initializePlatformSpecificAOTRelocationHeader which must be implemented on all
-    *        platforms that support AOT.
+    * @brief Initializes the common fields in the raw relocation record header.
     *
     * @param relocation pointer to the iterated external relocation
     * @return pointer into the buffer right after the fields of the header (ie the offsets section)
     */
    virtual uint8_t *initializeAOTRelocationHeader(TR::IteratedExternalRelocation *relocation);
 
-   void initializePlatformSpecificAOTRelocationHeader(TR::IteratedExternalRelocation *relocation, TR_RelocationTarget *reloTarget, TR_RelocationRecord *reloRecord, uint8_t targetKind)
-      { TR_ASSERT_FATAL(false, "Should not be called!\n"); return; }
+   /**
+    * @brief Initialization of relocation record headers for whom data for the fields are acquired
+    *        in a manner that is specific to a particular platform. This is the default implementation.
+    *
+    * @param relocation pointer to the iterated external relocation
+    * @param reloTarget pointer to the TR_RelocationTarget object
+    * @param reloRecord pointer to the associated
```

Comment:
Seems something is missing after "associated." I also see this phrasing in the platform-specific headers, so I guess it was copied from there

Come to think of it, I don't think it really helps anything for the doc comment to be duplicated. If you agree, maybe replace the platform-specific copies with a one-line note pointing to here

Reply:
Done in [force push](https://github.com/eclipse-openj9/openj9/compare/6a8bd2a1f7ba7914cf66c88fca7ab123435c12fc..3ae2f331966f06beab38c71248952813bcac3f04)